pci/src/vfio.rs

```diff
@@ -555,15 +555,15 @@ impl VfioPciDevice {
                 }
 
                 let slot = mem_slot();
-                let mem_region = vm.make_user_memory_region(
+                let mem_region = vm.lock().unwrap().make_user_memory_region(
```

Comment:
Add `let vm = vm.lock().unwrap();` at the beginning of the function then you don't need to modify the rest of the function.

Actually I think this makes more sense because you will want the lock to be held for the entirety of this function, no?

Reply:
Added a local variable to the top of the function, made sense to acquire a single lock to vm at the top of the function and reuse it.